### ISSUE TITLE
Executing code action, "Mark as Untainted" gives a compilation error

### ISSUE THREAD

**Issue by rasika** [ballerina-platform/ballerina-lang#18984]
**Description:**
Executing code action, "Mark as Untainted" gives a compilation error.

**Steps to reproduce:**
```ballerina
import ballerina/http;
import ballerina/io;

public function main() returns error? {
    io:println("Running ..");
    http:Client hc = new ("https://api.ipify.org");
    var res = check hc->get("/?format=json");
    var jpl = check res.getJsonPayload();
    var ipjson = check jpl.ip;
    io:println("My public IP is: ", ipjson);
    string ipaddr = ipjson.toJsonString();

    http:Client hc2 = new ("https://geo.ipify.org");
    res = check hc2->get("/api/v1?apiKey=&ipAddress=" + ipaddr);
    io:println(res.getJsonPayload());
}
```

**Affected Versions:**
1.0.0-RC

**Comment by kaviththiranga** [comment]
Fixed with Fix untainted code action #18993

**Comment by kaviththiranga** [comment]
Merged https://github.com/ballerina-platform/ballerina-lang/pull/18993
